content/source/docs/cloud/sentinel/sentinel-tf-012.html.md

```diff
@@ -170,6 +184,36 @@ namespace](./import/tfplan.html#namespace-resource-diff)
 to validate whether or not a value is unknown before looking for it in
 `applied`.
 
+### Changes Affecting Resources Being Destroyed but not Re-created
+
+In Terraform 0.11, when a resource is being destroyed but not re-created, it's
+[`diff`](./import/tfplan.html#value-diff) value in the `tfplan` import is empty
+while its [`applied`](./import/tfplan.html#value-applied) value has data. In
```

Comment:
We have tests indicating that `applied` should be empty even on TF 0.11. Let's discuss this offline.

Reply:
As we discussed offline, applied no longer shows up during destroy, even in TF 0.11 due to a change in a PR in the sentinel-plugin-terraform rep. I will remove the reference to that.